src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Octokit;
+
+namespace Microsoft.Docs.Build
+{
+    internal class GitHubAccessor
+    {
+        private readonly GitHubClient _client = new GitHubClient(new ProductHeaderValue("DocFXv3"));
+        private readonly object _syncRoot = new object();
+        private bool _isRateLimitExceeded = false;
+
+        public async Task<(List<Error> errors, UserProfile profile)> GetUserProfileByName(string name)
```

Comment:
>GetUserProfileByName [](start = 69, length = 20)

what will happened if it returns empty errors with null profile?

Reply:
when the user name doesn't exist on GitHub.

---
In reply to: [207148847](https://github.com/dotnet/docfx/pull/3144#discussion_r207148847) [](ancestors = 207148847)

Reply:
merge this to autho not found warning

---
In reply to: [207151809](https://github.com/dotnet/docfx/pull/3144#discussion_r207151809) [](ancestors = 207151809,207148847)